src/components/ToDo.js

```diff
@@ -65,11 +106,14 @@ const ToDo = () => {
             inputText={inputText}
             handleChange={handleChange}
             handleSubmit={handleSubmit}
+            editingItem={editingItem}
+            handleEdit={handleEdit}
```

Comment:
Event props definition should be prefix with `on`
```suggestion
            onEdit={handleEdit}
```